src/main/java/TaskManager.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class TaskManager {
+    public static void main(String[] args) {
+        Task[] taskList = new Task[3];
+        taskList[0] = new Task("read book");
+        taskList[1] = new Task("return book");
+        taskList[2] = new Task("buy bread");
+
+        taskList[0].markAsDone();
+        
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        while(!input.equals("bye")){
+            if(input.equals("list")){
+                System.out.println("Here are the tasks in your list:");
+                for(int i =0 ; i<3; i++){
+                    System.out.println((i+1) + ". " + taskList[i].toString());
+                }
+            }
+            else if(input.contains("unmark")){
```

Comment:
Consider else / else if on the same line after the }. 
Consider the spaces between ) and {.